src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Component/DefaultIcon.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Auth.Web.Admin.Rcl.Pages.Component;
+
+public class DefaultIcon : MIcon
+{
+    [Parameter]
+    public bool Add { get; set; }
+
+    [Parameter]
+    public bool Update { get; set; }
+
+    [Parameter]
+    public bool Remove { get; set; }
+
+    [Parameter]
+    public bool Authorize { get; set; }
+
+    protected override void OnInitialized()
+    {
+        if ((Small, Large, XLarge, XSmall) == (false, false, false, false)) Small = true;
+        if (Color is null) Color = "default-regular";
+        string icon = "";
+        if (Add) icon = "mdi-plus";
```

Comment:
Enum? and EnumDescription?

Reply:
This component has been removed in the new branch